diode-server/gen/dbstore/postgres/adapters.go

```diff
@@ -0,0 +1,40 @@
+package postgres
+
+import (
+	"fmt"
+
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"github.com/netboxlabs/diode/diode-server/gen/diode/v1/diodepb"
+	"github.com/netboxlabs/diode/diode-server/gen/diode/v1/reconcilerpb"
+)
+
+// ToProtobuf converts sqlc structure to analogous protobuf
+func (log IngestionLog) ToProtobuf() (*reconcilerpb.IngestionLog, error) {
```

Comment:
Could we use more idiomatic shorted naming (quite commonly used AFAIK)
```suggestion
// ToProto converts sqlc structure to analogous protobuf
func (log IngestionLog) ToProto() (*reconcilerpb.IngestionLog, error) {
```